security-center/snippets/v2/updateBigQueryExport.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ *  Updates an existing BigQuery export.
+ */
+function main(organizationId, exportId, dataset, location = 'global') {
+  // [START securitycenter_update_bigquery_export_v2]
+  // Imports the Google Cloud client library.
+  const {SecurityCenterClient} = require('@google-cloud/security-center').v2;
+
+  // Creates a new client.
+  const client = new SecurityCenterClient();
+  /**
+   *  Required. Name of the BigQuery export to retrieve. The following list shows
+   *  some examples of the format:
+   *  +
+   *  `organizations/{organization}/locations/{location}/bigQueryExports/{export_id}`
+   *  + `folders/{folder}/locations/{location}/bigQueryExports/{export_id}`
+   *  + `projects/{project}locations/{location}/bigQueryExports/{export_id}`
+   */
+  const name = `organizations/${organizationId}/locations/${location}/bigQueryExports/${exportId}`;
```

Comment:
issue: Referencing organizationId and location from outside the region tag scope is a violation of the copy-paste-runnable goal for samples.

suggestion: Minimally, any referenced variable should have a commented out variable declaration with an instruction for use on each variable.

For example,

```
  /**
   * TODO(developer): Uncomment the following line before running the sample.
   */

  // const projectId = 'YOUR_PROJECT_ID';
```

In https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/security-center/snippets/v2/createFinding.js it appears the decision was made to hard-code location to global. If that's how we're testing, hardcoding the global value without comment and adding an inline comment explaining would be better than a commented out instantiation.

Reply:
Addressed.